### ISSUE TITLE
enhancement: gtk context menu for root window on right click

### ISSUE THREAD

**Issue by acrisci** [i3/i3#1475]
Right clicking on the root window or i3bar should open a context menu that displays options for the user to run common commands such as opening a terminal, navigating the tree, editing the config, and switching workspaces.

**Comment by stapelberg** [comment]
A terminal is only useful when using the keyboard, so you might as well open it using a keyboard shortcut.

Switching workspaces already works by clicking on the workspace number, there is no need for another mechanism.

Editing the config is not supposed to be something that you do often, and i3-nagbar already prompts you to when you have a mistake in it. When you’re not pressed (no mistake in the config), you might as well just open the config file in your favorite editor, which you typically have handy already without needing another shortcut.

Navigating the tree is either done by clicking or using the key bindings, so I don’t see the point of having a menu for this.

In conclusion: I’m not convinced this is useful. If anything, this should be a third-party project, I think.
